BuildLoader.py

```diff
@@ -152,6 +152,8 @@ def __init__(self, *args, **kwargs):
 		self.VERINFO_SVN            = 1
 		self.VERINFO_BUILD_DATE     = '01/01/2018'
 
+		self.LOWEST_SUPPORTED_FW_VER   = ((self.VERINFO_PROJ_MAJOR_VER << 8) + self.VERINFO_PROJ_MINOR_VER)
```

Comment:
You will need to set the default value here and override this in your BoardConfig.py.  Since hte lowest supported FW ver is project specific,  project will need to maintain that.

Reply:
Done